eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/components/soap/SOAPBindingReferenceComposite.java

```diff
@@ -202,6 +203,11 @@ public void widgetSelected(final SelectionEvent e) {
         upChxGD.horizontalSpan = 3;
         _unwrappedPayloadCheckbox.setLayoutData(upChxGD);
 
+        _copyNamespacesCheckbox = createCheckbox(composite, "Copy Namespaces");
+        GridData cnChxGD = new GridData(GridData.FILL_HORIZONTAL);
```

Comment:
this is fine like that too but I would suggest to use GridDataFactory.fillDefaults().span(3,1).create();
I like the usage of the Factory which is providing a Fluent API